src/plugins/buttons/player-emotes-plugin.ts

```diff
@@ -117,9 +147,15 @@ export const action: buttonAction = (details) => {
     const { player, buttonId } = details;
 
     const emote = emotes[buttonId];
-
+    
     if(emote.name === 'SKILLCAPE') {
-        player.sendMessage(`You need to be wearing a skillcape in order to perform that emote.`);
+        if (skillcapeEmotes.some(item => item.itemIds.includes(player.getItemInEquipmentSlot('BACK')?.itemId))) {
```

Comment:
not really important, but a check if wearing any cape first might be slightly faster?

Reply:
Added a check as requested